controls/treeview/descriptor.md

```diff
@@ -0,0 +1,84 @@
+---
+title: TreeView Descriptor
```

Comment:
If I get the idea correctly, the main purpose of the `TreeViewDescriptor` class is to control the hierarchy and where in the hierarchy an item gets displayed. If this is the case, consider a title that is more use-case-centric instead of class-name-centric. If my assumption is wrong, ignore the title-related suggestions in this review.

```suggestion
title: Item Hierarchy
```